packages/core/src/queries/connector.ts

```diff
@@ -14,6 +14,13 @@ export const findAllConnectors = async () =>
     from ${table}
   `);
 
+export const findConnectorsByIds = async (ids: string[]) =>
+  pool.many<Connector>(sql`
```

Comment:
better add a check on `ids.length` and result length.

Reply:
will do it in #412

Reply:
Won't fix, since no longer need `findConnectorsByIds`. 
See the [comment](https://github.com/logto-io/logto/pull/412#issuecomment-1072058138) in #412